src/main/java/md/miller1995/Dealership/models/entities/CarEntity.java

```diff
@@ -13,6 +13,7 @@
 @NoArgsConstructor
 @AllArgsConstructor
```

Comment:
It's not such an important remark, but the most important annotations should be the first ones (closer to the class declaration).

Reply:
- updated